client/src/holdings/HoldingsList.tsx

```diff
@@ -167,14 +171,21 @@ export function HoldingsList({user, fxRates}: IProps) {
       setTotalPerformance(getTotalPerformance(holdings));
       setHoldings([...holdings]);
     } catch (err) {
-      console.error(err);
+      showNotification(
+        `Delete transaction ${holdings[holdingIndex].symbol} failed!`,
```

Comment:
Can we use the `isTrade` function to use the currect terminology. 

```ts
const transactionType = isTrade(t) ? 'trade' : 'dividend';
showNotification(
  `Delete ${transactionType} ${holdings[holdingIndex].symbol} failed!`,
  'error'
);
```

Also, as we're not doing this `holdings[holdingIndex].symbol` three times in this function, can you do this first at the top of the function?

`const holding = holdings[holdingIndex];`

Then after that you can just use `holding.symbol` instead of `holdings[holdingIndex].symbol`.